instrumentation/logback/logback-mdc-1.0/library/src/main/java/io/opentelemetry/instrumentation/logback/mdc/v1_0/OpenTelemetryAppender.java

```diff
@@ -102,20 +104,14 @@ private void processEvent(ILoggingEvent event) {
                   "baggage." + key, value.getValue()));
     }
 
-    if (eventContext == null) {
-      eventContext = contextData;
-    } else {
-      eventContext = new UnionMap<>(eventContext, contextData);
```

Comment:
was there some benefit in retaining both individual maps with the use of `UnionMap` that we lose now?

Reply:
It avoided copying the original map.